mama/c_cpp/src/c/dqstrategyplugin/dqstrategyplugin.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * mamaPluginTemplate.h
+ *
+ *  Created on: 23 Mar 2015
+ */
+
+#ifndef MAMAPLUGINHOOK_H_
+#define MAMAPLUGINHOOK_H_
+
+#include "mama/mama.h"
+#include "mama/status.h"
+#include "mama/types.h"
+#include "dqstrategy.h"
+
+#if defined( WIN32 )
+#   define PLUGINExpDLL __declspec( dllexport )
+#else
+#   define PLUGINExpDLL
+#endif
+#if defined(__cplusplus)
+extern "C" {
+#endif
+
+PLUGINExpDLL
+mama_status
+dqstrategyMamaPlugin_initHook (mamaPluginInfo* pluginInfo);
+
+PLUGINExpDLL
+mama_status
+dqstrategyMamaPlugin_transportPostCreateHook (mamaPluginInfo pluginInfo, mamaTransport transport);
+
+PLUGINExpDLL
+mama_status
+dqstrategyMamaPlugin_publisherPreSendHook (mamaPluginInfo pluginInfo, mamaPublisher publisher, mamaMsg message);
+
+PLUGINExpDLL
+mama_status
+dqstrategyMamaPlugin_transportEventHook (mamaPluginInfo pluginInfo, mamaTransport transport, int setStale, mamaTransportEvent tportEvent);
```

Comment:
A lot of these lines are pretty significantly beyond the 80 character mark, which makes them harder to read. You should be able to run these files through the clang-format rules and get something that matches the style guide, but for reference it's discussed here: https://openmama.github.io/openmama_coding_standards.html#line-width